admin/client.py

```diff
@@ -180,26 +168,19 @@ class RunOptions(Options):
         ['distribution', None, None,
          'The target distribution. '
          'One of {}.'.format(', '.join(DISTRIBUTIONS))],
-        ['provider', None, 'rackspace',
-         'The target provider to test against. '
-         'One of {}.'.format(', '.join(PROVIDERS))],
-        ['config-file', None, None,
-         'Configuration for providers.'],
         ['branch', None, None, 'Branch to grab packages from'],
         ['flocker-version', None, flocker.__version__,
          'Version of flocker to install'],
-        ['flocker-version', None, flocker.__version__,
-         'Version of flocker to install'],
         ['build-server', None, 'http://build.clusterhq.com/',
          'Base URL of build server for package downloads'],
-    ]
-
-    optFlags = [
-        ["keep", "k", "Keep VMs around, if the tests fail."],
+        # XXX - remove the remaining flags once Buildbot is updated
```

Comment:
OK, although there is no JIRA issue for removing, maybe I should identify the "once Buildbot is updated" to point to FLOC-2813.